From 21e340cde2c62f35fd04b5f4c576432f503472c6 Mon Sep 17 00:00:00 2001
From: Sam Edwards <CFSworks@gmail.com>
Date: Fri, 28 Jul 2023 15:31:29 -0600
Subject: [PATCH] buildroot: HACK; don't overwrite .cargo/config!

Rebased onto Buildroot 2025.02.17: the surrounding block gained a
quoted "${manifest}" variable, so the old hunk only landed with fuzz.

---
 support/download/cargo-post-process | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/support/download/cargo-post-process
+++ b/support/download/cargo-post-process
@@ -38,7 +38,7 @@
 cargo vendor \
     --manifest-path "${manifest}" \
     --locked VENDOR \
-    > .cargo/config
+    >> .cargo/config
 
 # "cargo vendor' outputs on stderr a message directing to add some data
 # to the project's .cargo/config.toml, data that it outputs on stdout.
-- 
2.43.0
